packages/plugins/urlWriteback/index.ts

```diff
@@ -0,0 +1,45 @@
+import { updateEntity } from "./yext.ts";
+
+/**
+ * The shape of the configuration options for the writeback plugin
+ */
+export interface UrlWritebackConfig {
+  field: string
```

Comment:
Colons. The formatter and linter should catch these things. You might need to rebase if you haven't in awhile.